cli/src/pcluster/validators/iam_validators.py

```diff
@@ -57,6 +59,28 @@ def _validate(self, policy: str):
             self._add_failure(str(e), FailureLevel.ERROR)
 
 
+class IamResourcePrefixValidator(Validator):
+    """
+    Iam Resource Prefix Validator.
+
+    Verify if the Resource Prefix is compliant with IAM naming conventions
+    """
+
+    IAM_PATH_NAME_REGEX = r"^((/[a-zA-Z0-9_.,+@=-]+)+)/"
+    IAM_ROLE_NAME_PREFIX = r"^[a-zA-Z0-9_.,+@=-]+$"
+
+    def _validate(self, resource_prefix: str):
+        if not (
+            re.match(IamResourcePrefixValidator.IAM_PATH_NAME_REGEX, resource_prefix)
+            or re.match(IamResourcePrefixValidator.IAM_ROLE_NAME_PREFIX, resource_prefix)
+        ):
+            self._add_failure(
+                f"Unsupported format for ResourcePrefix {resource_prefix} "
+                f"Please, refer to our public docs for further details.",
```

Comment:
1. I'd say "our official documentation" instead of "public docs" because the latter sounds too "internal".
2. Missing full stop at the end of the first sentence.

Reply:
Sure